src/library/dao/cue.h

```diff
@@ -46,13 +47,16 @@ class Cue : public QObject {
     QString getLabel() const;
     void setLabel(const QString label);
 
+    QColor getColor() const;
+    void setColor(const QColor color);
```

Comment:
It is uncommon to use a const qualifier for function parameters that are passed by value. I remove those redundant qualifiers from existing code when I stumble over them.